Dungeoneer/assets/data/entities.dat

```diff
@@ -1,5 +1,10 @@
 {
 	entities:{
+	    "Features": {
+			"NeoText": {
+				class: com.interrupt.dungeoneer.entities.NeoText
+			}
+	    }
```

Comment:
Think there's a missing comma here for this to be valid json

Reply:
Correct, the parser did not pick it up. Fixed in cc939d3.